packages/moonstone/Image/Image.js

```diff
@@ -62,6 +64,57 @@ const ImageBase = kind({
 	}
 });
 
+
+// This induces a render when there is a screen size change that has a corosponding image src value
+// associated with the new screen size. The render is kicked off by remembering the new image src.
+//
+// This hoc could (should) be rewritten at a later time to use a smarter context API and callbacks,
+// or something like pub/sub; each of which would be hooked together from the resolution.js that
+// would coordinate any screen size/orientation changes and emit events from there.
+//
+// This is ripe for refactoring, and could probably move into UI to be generalized, but that's for
+// another time. -B 2018-05-01
+const ResponsiveImageDecorator = hoc((config, Wrapped) => {
```

Comment:
Should this same decorator be applied to `GridlistImageItem`?

Reply:
I suppose it could.